### ISSUE TITLE
eth-typing v2.0.0 version conflict

### ISSUE THREAD

**Issue by stellarspot** [singnet/snet-cli#58]
Integration test fails with exception:
https://circleci.com/gh/stellarspot/platform-pipeline/45
`pkg_resources.ContextualVersionConflict: (eth-typing 2.0.0 (/usr/local/lib/python3.6/dist-packages), Requirement.parse('eth-typing<2.0.0,>=1.3.0'), {'eth-utils'})`

dependency tree:
```
  - web3 [required: ==4.2.1, installed: 4.2.1]
    - eth-abi [required: >=1.1.0,<2, installed: 1.2.0]
      - eth-typing [required: <=2, installed: 2.0.0]
```

11 days ago eth-typing v2.0.0 was released.
There is the fix for the eth-abi dependency to use 'eth-typing<2' which has not been released yet:
https://github.com/ethereum/eth-abi/commit/04f1373750475d4158fb5b2713d6389f8d7fbd7c

**Comment by vsbogd** [comment]
Let's wait for @stellarspot's opinion

**Comment by stellarspot** [comment]
I suggest to check if it is possible to revert the fix #59 back because the [eth-abi](https://github.com/ethereum/eth-abi/releases) v2.0.0-beta.2 which has the necessary fix has been released.

**Comment by stellarspot** [comment]
I tried to remove the 'eth-typing<2' dependency and got the message:
`eth-abi 1.2.2 has requirement eth-typing<2, but you'll have eth-typing 2.0.0 which is incompatible.
`

**Comment by Chris-Ricketts** [comment]
Following the [How to Publish a SingularityNET Service Example 2 Tutorial](https://github.com/singnet/wiki/tree/master/tutorials/howToPublishService) I got a similar error executing `snet identity create test_id key`:
```
Traceback (most recent call last):
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 574, in _build_master
    ws.require(__requires__)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 892, in require
    needed = self.resolve(parse_requirements(requirements))
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 783, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (eth-typing 1.3.0 (/usr/local/lib/python3.6/dist-packages), Requirement.parse('eth-typing<3.0.0,>=2.0.0'), {'eth-abi'})

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "/usr/local/bin/snet", line 6, in <module>
    from pkg_resources import load_entry_point
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3088, in <module>
    @_call_aside
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3072, in _call_aside
    f(*args, **kwargs)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 3101, in _initialize_master_working_set
    working_set = WorkingSet._build_master()
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 576, in _build_master
    return cls._build_from_requirements(__requires__)
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 589, in _build_from_requirements
    dists = ws.resolve(reqs, Environment())
  File "/usr/lib/python3/dist-packages/pkg_resources/__init__.py", line 783, in resolve
    raise VersionConflict(dist, req).with_context(dependent_req)
pkg_resources.ContextualVersionConflict: (eth-typing 1.3.0 (/usr/local/lib/python3.6/dist-packages), Requirement.parse('eth-typing<3.0.0,>=2.0.0'), {'eth-abi'})
```

**Comment by astroseger** [comment]
closed by #145
